src/status_im2/contexts/quo_preview/list_items/token_value.cljs

```diff
@@ -0,0 +1,65 @@
+(ns status-im2.contexts.quo-preview.list-items.token-value
+  (:require
+    [quo2.core :as quo]
+    [react-native.core :as rn]
+    [reagent.core :as reagent]
+    [status-im2.contexts.quo-preview.preview :as preview]))
+
+(def descriptor
+  [{:label   "Token:"
+    :key     :token
+    :type    :select
+    :options [{:key   :eth
+               :value "ETH"}
+              {:key   :snt
+               :value "SNT"}]}
+   {:label   "State:"
+    :key     :state
+    :type    :select
+    :options [{:key   :default
+               :value "Default"}
+              {:key   :pressed
+               :value "Pressed"}
+              {:key   :active
+               :value "Active"}]}
+   {:label   "Status:"
+    :key     :status
+    :type    :select
+    :options [{:key   :empty
+               :value "Empty"}
+              {:key   :positive
+               :value "Positive"}
+              {:key   :negative
+               :value "Negative"}]}
+   {:label   "Color:"
+    :key     :color
+    :type    :select
+    :options [{:key   :blue
+               :value "Blue"}
+              {:key   :orange
+               :value "Orange"}
+              {:key   :green
+               :value "Green"}]}
```

Comment:
I believe this will be the customization color. WDYT?

Reply:
@smohamedjavid Oh I didn't know we have a preview option for that. Thanks.